Makefile

```diff
@@ -0,0 +1,30 @@
+DOCKER=docker
+
+TAG=gh-pages
+
+# Build the docker image
+image:
+	${DOCKER} build -t ${TAG} .
+
+# Produce a bash shell
+shell:
+	${DOCKER} run --rm -it \
+		-p 4000:4000 \
+		-v ${PWD}:/gh-pages \
+		${TAG} \
+		/bin/bash
+
+# Spawn a server. Specify the path to the SITE directory by
+# exposing it using `expose SITE="../path-to-jekyll-site"` prior to calling or
+# by prepending it to the make rule e.g.: `SITE=../path-to-site make server`
+server:
+	ls "${SITE}" >/dev/null && \
+	${DOCKER} run --rm -it \
+		-p 4000:4000 \
+		-v ${PWD}:/gh-pages \
+		-v `realpath ${SITE}`:/site \
+		-w /site \
```

Comment:
Can we use `/src/site` (a subdirectory 2 levels deep) for reasons stated above?

Reply:
Done :wink: 9d8fe1886fccaf4355df158dac74707eb87cec37